modules/cli/src/test/kotlin/datamaintain/cli/update/db/UpdateDbTest.kt

```diff
@@ -350,9 +350,43 @@ internal class UpdateDbTest : BaseCliTest() {
                 }
             }
         }
+
+        @Nested
+        inner class Flags {
+            @Test
+            fun `should build config with one flag`() {
+                // Given
+                val updateDbArguments = listOf("--flags=MY_TEST_FLAG")
+
+                // When
+                runUpdateDb(updateDbArguments)
+
+                // Then
+                expectThat(configWrapper.datamaintainConfig!!.flags.toList()) {
```

Comment:
I can't see why there is a ```toList()``` here but no one in the test that comes after. 
To make it simpler, you could use ```containsExactly```.